package.json

```diff
@@ -14,12 +14,15 @@
     "start:debug": "cross-env NODE_ENV=development nest start --debug --watch",
     "start:prod": "node dist/main",
     "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
+    "lint:ci": "eslint ./src/",
```

Comment:
Seems the lint:ci script is not suitable for this project

Reply:
Do you mean this step should be removed or changed?